src/processors/modulation/CleanDelayType.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
I thought since this was being used by chorus and flanger now it could be in its own file. I noticed the delay module uses a similar version of this struct but initializes the delay line with a different value 1 << 20 instead of 1 << 18.

Should I look into making this struct more global and we can use it for the delay as well?